docs/user-guide/solver/09-command-line.md

```diff
@@ -18,7 +18,13 @@ hide:
 | --parallel               | Enable [parallel](optional-features/multi-threading.md) computation of MC years                                                                                       |
 | --force-parallel=VALUE   | Override the max number of years computed [simultaneously](optional-features/multi-threading.md)                                                                      |
 | --linear-solver=VALUE    | The optimization solver to use for linear problems. Possible values are: `sirius` (default, LP only), `coin`, `xpress`, `scip` (MIP only), `glpk` (Linux only), `highs`, `pdlp` (LP only) |
+| --linear-solver-parameters=VALUE | Linear solver-specific parameters. Syntax is solver-dependent, and only supported for `sirius` and `xpress` |
```

Comment:
Why not scip ?

Reply:
sorry : cmd line option is available for **scip** and **xpress**.
Now fixed.